packages/api/src/schema/types/user.ts

```diff
@@ -153,6 +153,7 @@ export default [
       allowedActions: [RoleAction!]!
       tokens: [Token!]!
       oAuthAccounts: [OAuthAccount!]!
+      impersonator: User
```

Comment:
Remark: The impersonator is actually not something that is unique to one user but actually based on the current session / login token used. Consider using a root level Query.impersonator instead of putting it on the actual user. Similar as Query.me